src/packages/media/media/property-editors/image-cropper/property-editor-ui-image-cropper.element.ts

```diff
@@ -29,21 +30,20 @@ export class UmbPropertyEditorUIImageCropperElement extends UmbLitElement implem
 		}
 	}
 
-	// #crops = [];
-
-	// @property({ attribute: false })
-	// public set config(config: UmbPropertyEditorConfigCollection | undefined) {
-	// 	this.#crops = config?.getValueByAlias('crops') ?? [];
+	@property({ attribute: false })
+	public set config(config: UmbPropertyEditorConfigCollection | undefined) {
+		const crops = config?.getValueByAlias<UmbImageCropperPropertyEditorValue['crops']>('crops') ?? [];
 
-	// 	if (!this.value) {
-	// 		//TODO: How should we combine the crops from the value with the configuration?
-	// 		this.value = {
-	// 			crops: this.#crops,
-	// 			focalPoint: { left: 0.5, top: 0.5 },
-	// 			src: 'https://picsum.photos/seed/picsum/1920/1080',
-	// 		};
-	// 	}
-	// }
+		if (!this.value) {
+			this.value = {
+				src: '',
+				crops: crops,
+				focalPoint: { left: 0.5, top: 0.5 },
+			};
+		} else {
+			this.value.crops = crops;
```

Comment:
Would this not mean that you are overriding any stored crops with the config by doing this? Do crops not also hold references to media items, or are they just simple config objects?